src/main/config.ts

```diff
@@ -35,6 +35,7 @@ export interface Configuration {
   password: string;
   callsign: string;
 
+  radioEffects: string;
```

Comment:
Since there are only three valid values for this they should be explicitly listed instead of just a string.

```suggestion
  radioEffects: "off" | "input" | "output";
```

Reply:
I guess it's four valid values? Include `on` in this as well.

Reply:
Yes, there are four values for the four flag combinations.